src/operator/tensor/slice_sum.cc

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file slice_sum.cc
+ * \brief Fused implementation of slice and sum operation
+ */
+#include "./slice_sum-inl.h"
+
+namespace mxnet {
+namespace op {
+
+DMLC_REGISTER_PARAMETER(SliceSumParam);
+
+NNVM_REGISTER_OP(_slice_sum)
```

Comment:
Do you intend to introduce a graph pass to automatically use this op when applicable? Otherwise starting its name with "_" makes it hidden from users.

Reply:
Not planned for now. I changed the op name to slice_sum.